README.md

```diff
@@ -26,3 +26,5 @@ Please complete the following by November 1st. This should be the next time we w
 * Trey Watford is Bester
 * Will Dixon Is better
 * Chris Bracey Is betterer
+* Kyle Heath Is bettererer
```

Comment:
https://github.com/fmuacm/textContributors/network

This is showing it was not merged in. Try

```
git checkout dylans
git fetch --prune
git merge origin/develop dylans
```

If you get merge requests just let me know so we can walk through it.
will.dixon@acstechnologies.com
We may jump into a google hangleout.